aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -134,13 +136,15 @@ struct TORCH_API DispatchKeyExtractor final {
         }
       }
     });
-    return dispatchKeySetToDispatchKey_(DispatchKeySet::FULL, ks);
+    // Keys that are fallthrough should be skipped
+    return impl::computeDispatchKeySet(ks, nonFallthroughKeys_);
```

Comment:
Is this a bugfix?

Reply:
I guess you just inlined dispatchKeySetToDispatchKey_ directly here

Reply:
yeah, just a piece of feedback that Basil left when I was trying to figure out a reasonable rename for `dispatchKeySetToDispatchKey_`. It technically doesn't have to go in this PR (if you can come up with a not-awful name)